src/bin/server.ts

```diff
@@ -32,6 +33,10 @@ interface Args {
     }
     wsPort = args["port"];
   }
+  let wsHost = "0.0.0.0";
+  if (args["host"]) {
+    wsHost = args["host"];
+  }
```

Comment:
We're now overriding the Z-Wave JS defaults and maintain our own for `host`. Previously, it would do whatever Z-Wave JS would do. I don't think it matters in this case, just something to keep in mind when we add future CLI options.